### DISCUSSION TITLE
Stats card not working

### DISCUSSION THREAD

**Discussion by Arturs1123** [DenverCoder1/github-readme-streak-stats#649] [Q&A] [answered]
Hello, everyone!

It's been several days since my stats card is not working properly and does not appear on my README file. I have checked all the settings and cannot find the issue.

Is there any issue with the project?

**Comment by Luis981123** [comment, marked as answer]
I have same problem.
Do you solve this problem?

**Comment by ikostan** [comment]
Hi all.

Same issue here.  See code i am using down below:
`---
### Github Streak Stats

[![GitHub Streak](https://streak-stats.demolab.com?user=ikostan&theme=dark&date_format=M%20j%5B%2C%20Y%5D)](https://git.io/streak-stats)

---
`
Screenshot:
![image](https://github.com/user-attachments/assets/8f611a22-39c6-41b2-9374-4b47cd9059dd)

Any help would be much appritiated.

**Comment by KarthiDreamr** [comment]
https://github-readme-streak-stats-eight.vercel.app/demo/

You should try this one!

@DenverCoder1 please add this one too in your readme as mirror link, 
or just ping me, I'll host and give you mirror link

**Comment by Tsedexashu08** [comment]
> Hello, everyone!
> 
> It's been several days since my stats card is not working properly and does not appear on my README file. I have checked all the settings and cannot find the issue.
> 
> Is there any issue with the project?

It's usual when there servers have to many requests it'll reappear if u wait a day or 2..that's what I did.
